test/e2e_node/gpus.go

```diff
@@ -151,6 +151,29 @@ var _ = framework.KubeDescribe("GPU [Serial]", func() {
 
 			By("Checking if the pod outputted Success to its logs")
 			framework.ExpectNoError(f.PodClient().MatchContainerOutput(podFailure.Name, podFailure.Name, "Success"))
+
+			By("checking allocatable gpus")
+			gpusAvailableBefore := getGPUsAvailable(f)
```

Comment:
Yes that would work. What I was hoping to do was check for any number and make sure that number exists after a kubelet restart.